app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx

```diff
@@ -59,7 +61,16 @@ export class Container extends React.Component<IContainerProps, IContainerState>
       dockerImages: cmd.backingData && cmd.backingData.filtered ? cmd.backingData.filtered.images : [],
       targetGroupMappings: cmd.targetGroupMappings,
       targetGroupsAvailable: cmd.backingData && cmd.backingData.filtered ? cmd.backingData.filtered.targetGroups : [],
+      containerMappings: cmd.containerMappings,
     };
+
+    this.state.targetGroupMappings.forEach(targetGroupMapping => {
+      targetGroupMapping.containerName = "";
+    });
+
+    this.state.containerMappings.forEach(containerMapping => {
+      containerMapping.containerName = "";
+    });
```

Comment:
You don't actually need to be setting containerMappings here.

What you should do is actually set cmd.containerMappings to null or an empty array of ContainerMappings. ContainerMappings are only used when taskDefArtifacts are used.